etc/ciao/ciao.json

```diff
@@ -0,0 +1,13 @@
+{
+    "launcher": {
+	"server": "localhost",
+	"cacert": "/etc/pki/ciao/CAcert-server-localhost.pem",
+	"cert": "/etc/pki/ciao/cert-client-localhost.pem",
```

Comment:
I suppose this is a sample, but we should still make it more relevant and actually match what ciao-cert generates:

"cacert": "/etc/pki/ciao/cert-Scheduler-ciao-ctl.intel.com.pem"
"cert": "/etc/pki/ciao/cert-CNAgent-localhost.pem"